src/mlpack/methods/kde/kde_rules.hpp

```diff
@@ -44,8 +49,13 @@ class KDERules
            arma::vec& densities,
            const double relError,
            const double absError,
+           const double MCProb,
+           const size_t initialSampleSize,
+           const double MCAccessCoef,
+           const double MCBreakCoef,
```

Comment:
The variables here should also be `mcProb` (or however you want to do it, so long as the first letter is lowercase in accordance with the style guide), and a few other places around the file.